src/bool.cr

```diff
@@ -55,4 +55,9 @@ struct Bool
   def to_s(io)
     io << to_s
   end
+
+  # Returns `self`
```

Comment:
But this is for `clone`, I'm not sure we should refer to `dup`. I think `clone` for primitive types is expected to return the same value. Maybe it's better to just mark it as `:nodoc:`? "Returns `self`" doesn't look bad to me.